crates/ruff_python_formatter/src/lib.rs

```diff
@@ -552,4 +579,54 @@ if [
             writeln!(f)
         }
     }
+
+    #[derive(serde::Deserialize)]
```

Comment:
Would adding an optional `Deserialize` and `#[serde(default)]`  to `PyFormatOptions` also work? This would avoid keeping them in sync

Reply:
Oh nice!